.github/workflows/rust.yml

```diff
@@ -17,49 +17,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04]
+        os: [ubuntu-latest]
         rust: [stable]
         target: ["default"]
     steps:
     - uses: actions/checkout@v2
-    - name: Install deps on linux
-      if: startsWith(matrix.os, 'ubuntu')
-      run: |
-        sudo bash -c 'apt-add-repository -y https://mirrors.kernel.org/ubuntu'
-        sudo apt update
-        sudo apt install -y \
-          gstreamer1.0-libav \
-          gstreamer1.0-plugins-bad \
-          gstreamer1.0-plugins-base \
-          gstreamer1.0-plugins-good \
-          gstreamer1.0-plugins-ugly \
-          gstreamer1.0-tools \
-          libasound2-plugins \
-          libfaad2 \
-          libffi7 \
-          libfftw3-single3 \
-          libges-1.0-dev \
-          libgstreamer-plugins-bad1.0-dev\
-          libgstreamer-plugins-base1.0-dev \
-          libgstreamer1.0-dev \
-          libgstrtspserver-1.0-dev \
-          libjack-jackd2-0 \
-          libmms0 \
-          libmpg123-0 \
-          libopus0 \
-          liborc-0.4-0 \
-          liborc-0.4-dev \
-          libpulsedsp \
-          libsamplerate0 \
-          libspeexdsp1 \
-          libtdb1 \
-          libtheora0 \
-          libtwolame0 \
-          libwayland-egl1-mesa \
-          libwebrtc-audio-processing1 \
-          pulseaudio \
-          pulseaudio-utils \
-          webp
+    - name: Install dependencies
+      run: sudo apt-get update && sudo apt-get install -y libasound2-dev libjack-dev libpulse-dev ninja-build gcc
```

Comment:
I would recommend keeping around the various plugin deps, with development packages: libfaad-dev libmpg123-dev etc. That will keep the build time low, and also ensure that you build the plugins you expect. Not everything has wrap files, for example wayland and webp.

Just make sure you don't accidentally pull in gstreamer itself using one of the deps. I would place an assert after this checking that `libgstreamer-1.0.so.*` is not in libdir.